chapter_3/automation_using_ansible.md

```diff
@@ -1,11 +1,40 @@
 # Automation Using Ansible
 
-- Intro to Ansible
-- ManageIQ and Ansible
-- Ansible Terms
-  - Modules
-  - Playbooks
-  - Roles
-  - Inventory
-  - Job and Job Template
-- Ruby and Ansible
+## Introduction to Ansible
+
+Ansible is an opensource tool which serves for configuration management, orchestration, provisioning and deployments. It is developed using python and runs on Windows, Mac, and UNIX-like systems. The Ansible itself is agentless i.e you don't need to install any client side software.It uses SSH connections. Hence,if you have well-oiled SSH setup, then you are roll up to use ansible in your environment. This also means that you can install it on only one machine and control your infrastructure from that machine itself.
```

Comment:
Use "Open source" instead of "opensource"

Reply:
Done.